api/envoy/api/v2/core/base.proto

```diff
@@ -5,7 +5,6 @@ package envoy.api.v2.core;
 import "envoy/api/v2/core/address.proto";
 import "envoy/api/v2/core/backoff.proto";
 import "envoy/api/v2/core/http_uri.proto";
-import "envoy/type/percent.proto";
```

Comment:
If we go this route there are a few things:
1) I think there is some special way this needs to happen to not break compilation, like we move the file, but import it somehow, and I think we shouldn't change the package path (so it would still be core). @htuch can you advise?
2) Stepping back though (and sorry for suggesting this after you did so much work), I'm not sure we actually need to have per regex config for this. I wonder if it would be sufficient to just have a global / well known warn/error runtime key for all regex? I think that would be a lot simpler and likely sufficient. @jmarantz wdyt?
3) If we do go for config and require the package movement, can we split that out into a separate PR?

Thank you!

/wait

Reply:
1. `import public`
2. I think global is better here. Common base types shouldn't depend on core config.